ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-access-lists.j2

```diff
@@ -0,0 +1,124 @@
+{%                     if acle.destination_ports is arista.avd.defined %}
+{%                         set acl_entry.string = acl_entry.string ~ acle.destination_ports_match | arista.avd.default('eq') ~ ' ' %}
+{%                         for a_port in acle.destination_ports %}
+{%                             set acl_entry.string = acl_entry.string ~ a_port ~ ' ' %}
+{%                         endfor %}
+{%                     endif %}
+{%                 endif %}
+{#                 * add tcp flags #}
+{%                 if acle.protocol | lower == 'tcp' %}
+{%                     if acle.tcp_flags is arista.avd.defined %}
+{%                         for a_flag in acle.tcp_flags %}
+{%                             set acl_entry.string = acl_entry.string ~ a_flag ~ ' ' %}
+{%                         endfor %}
+{%                     endif %}
+{%                 endif %}
+{#                 * add icmp message type and code #}
+{%                 if acle.protocol | lower == 'icmp' %}
+{%                     if acle.icmp_type is arista.avd.defined %}
+{%                         set acl_entry.string = acl_entry.string ~ acle.icmp_type ~ ' ' %}
+{%                         if acle.icmp_code is arista.avd.defined %}
+{%                             set acl_entry.string = acl_entry.string ~ acle.icmp_code ~ ' ' %}
+{%                         endif %}
+{%                     endif %}
+{%                 endif %}
+{#                 * add nexthop group #}
+{%                 if acle.nexthop_group is arista.avd.defined %}
+{%                     set acl_entry.string = acl_entry.string ~ 'nexthop-group ' ~ acle.nexthop_group ~ ' ' %}
+{%                 endif %}
+{#                 * add match non-head fragment packets #}
+{%                 if acle.fragments is arista.avd.defined(true) %}
+{%                     set acl_entry.string = acl_entry.string ~ 'fragments ' %}
+{%                 endif %}
+{#                 * add tracked #}
+{%                 if acle.tracked is arista.avd.defined(true) %}
+{%                     set acl_entry.string = acl_entry.string ~ 'tracked ' %}
+{%                 endif %}
+{#                 * add ttl #}
+{%                 if acle.ttl is arista.avd.defined %}
+{%                     set acl_entry.string = acl_entry.string ~ 'ttl ' ~ acle.ttl_match | arista.avd.default('eq') ~ ' ' %}
+{%                     set acl_entry.string = acl_entry.string ~ acle.ttl ~ ' ' %}
+{%                 endif %}
+{#                 * add dscp #}
+{%                 if acle.dscp is arista.avd.defined %}
+{%                     set acl_entry.string = acl_entry.string ~ 'dscp ' ~ acle.dscp | lower ~ ' ' %}
+{%                 endif %}
+{#                 * add log #}
+{%                 if acle.log is arista.avd.defined(true) %}
+{%                     set acl_entry.string = acl_entry.string ~ 'log' %}
+{%                 endif %}
+{%             endif %}
+{#             - always remove trailing spaces at the end of the string #}
```

Comment:
In all other templates we try to implement spacing by adding the space _before_ the keyword you add. This way we ensure that the `acl_entry.string` is always clean and ready to be printed. Please adjust here and remove the rstrip

Reply:
closing, as that is discussed in the rstrip related comment above